src/reducers/books.js

```diff
@@ -0,0 +1,28 @@
+const intitialState = [];
+
+const CREATE_BOOK = 'CREATE_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```

Comment:
**[Required]** These are action types. We place it in the actions directory. Either place them inside the index.js file or create a new one for types only.